che-mount/entrypoint.sh

```diff
@@ -51,7 +78,8 @@ parse_command_line () {
 
   # See if profile document was provided
   mkdir -p $HOME/.unison
-  cp -rf /profile/default.prf $HOME/.unison/default.prf
+  
+  [ -f /profile/default.prf ] && cp -rf /profile/default.prf $HOME/.unison/default.prf
```

Comment:
I wouldn't introduce a newline as it means the line is not anymore part of the comment `# See if profile document was provided` but from another instruction